src/org/javarosa/core/model/utils/QuestionPreloader.java

```diff
@@ -109,8 +109,10 @@ public boolean handlePostProcess(TreeElement node, String params) {
             public String preloadHandled() {
                 return "uid";
             }
+
+            // Generates an RFC 4122 UUID with a "uuid:" prefix
             public IAnswerData handlePreload(String preloadParams) {
-                return new StringData(PropertyUtils.genGUID(25));
+                return new StringData("uuid:" + PropertyUtils.genUUID());
```

Comment:
Excuse my probably naive question but, why not `UUID.randomUUID().toString()`?

Reply:
Good question! I was just matching the `uuid()` implementation. I think that it predates Java 5 so that's likely the original reason for a custom method. I can change both if you prefer.

Reply:
Don't worry. I was supposing something on that line.

It looks like a good first issue for someone else ;)